docs/csharp/language-reference/keywords/value-types.md

```diff
@@ -9,26 +9,16 @@ helpviewer_keywords:
   - "C# language, value types"
 ms.assetid: 471eb994-2958-49d5-a6be-19b4313f80a3
 ---
-# Value Types (C# Reference)
-The value types consist of two main categories:  
-  
--   [Structs](../../../csharp/language-reference/keywords/struct.md)  
-  
--   [Enumerations](../../../csharp/language-reference/keywords/enum.md)  
-  
- Structs fall into these categories:  
-  
--   Numeric types  
-  
-    -   [Integral types](../../../csharp/language-reference/keywords/integral-types-table.md)  
-  
-    -   [Floating-point types](../../../csharp/language-reference/keywords/floating-point-types-table.md)  
-  
--   [bool](../../../csharp/language-reference/keywords/bool.md)  
-  
--   User defined structs.  
-  
-## Main Features of Value Types  
+# Value types (C# Reference)
+
+The value types consist of two main categories:
+
+- [Structs](struct.md)
+
+- [Enumerations](enum.md)
+
+## Main features of value types
+
  Variables that are based on value types directly contain values. Assigning one value type variable to another copies the contained value. This differs from the assignment of reference type variables, which copies a reference to the object but not the object itself.  
```

Comment:
I am having trouble with the "variables that are based on" phrase. How about something like:

> Variables in C# are declared or inferred to be a certain type. This type is is either a value or reference type. Variables that are a value type, like `int`, contain the value of the type, like `42`. This differs from variables that are a reference type, like `string`, which contain a reference (the memory location) to a value (otherwise known as an "object").